charts/consul/templates/server-acl-init-job.yaml

```diff
@@ -168,7 +168,7 @@ spec:
           CONSUL_FULLNAME="{{template "consul.fullname" . }}"
 
           consul-k8s-control-plane server-acl-init \
-            -log-level={{ .Values.global.logLevel }} \
+            -log-level={{ default .Values.global.logLevel .Values.server.logLevel}} \
```

Comment:
Does it make sense to have server-acl-init, server-acl-init-cleanup and tls-init use the `server.logLevel` value? The `server` values stanza is typically meant for configuring the consul servers themselves, and this value is now configuring a few consul-k8s jobs but right now it doesn't configure the consul servers themselves. Maybe for the server-acl-init and cleanup jobs, the log level could be under `.Values.acls`, and for tls-init the log level could be under `.Values.tls`.

Reply:
Makes sense. Removed `server.logLevel` and added ` global.acls.logLevel` and `global.tls.logLevel` instead